options/options.js

```diff
@@ -1,40 +1,54 @@
 "use strict";
 
 $(() => {
-  $('.message a').click(function(){
+  $('.message a').click(function(e){
     $('.error').hide();
-    $('.login-container').animate({height: 'toggle', opacity: 'toggle'}, 'slow');
+    $('.login-container').animate({height: 'hide', opacity: 'hide'}, 'slow');
+    $(`.${e.target.id}-login-container`).animate({height: 'show', opacity: 'show'}, 'slow');
   });
   $('#login').click((e) => {
     addCred(getGithubParam());
   });
   $('#ghe-login').click((e) => {
     addCred(getGHEParam());
   });
+  $('#bitbucket-login').click((e) => {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/leonhartX/gas-github/blob/ca0e019943cee791765d7c6379c60ec7ac8e41b1/options/options.js#L15"></span>‘e’ is defined but never used.</td>
    </tr>
  </tbody>
</table>